src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -48,12 +48,13 @@ public static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog, s
         /// <param name="catalog"> The transform's catalog.</param>
         /// <param name="outputColumnName"> Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName"> Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="colors"> Specifies which <see cref="ImagePixelExtractingEstimator.ColorBits"/> to extract from the image. The order of colors is: Alpha, Red, Green Blue.</param>
-        /// <param name="interleave">Wheather to interleave the pixels, meaning keep them in the `ARGB ARGB` order, or leave them separated in the planar form, where the colors are outputed one by one
-        /// alpha, red, green, blue for all the pixels of the image. </param>
-        /// <param name="scale">Scale color pixel value by this amount.</param>
-        /// <param name="offset">Offset color pixel value by this amount.</param>
-        /// <param name="asFloat">Output the array as float array. If false, output as byte array.</param>
+        /// <param name="colors">What colors to extract.</param>
+        /// <param name="order">In which order extract colors from pixel.</param>
+        /// <param name="interleave">Whether to interleave the pixels colors, meaning keep them in the <paramref name="order"/> order, or leave them in the plannar form:
+        /// first output one color values for all pixels, then another color and so on.</param>
```

Comment:
>one color values for all pixels [](start = 25, length = 31)

Maybe: "all the values for one color for all pixels, then all values for another color and so on."